### ISSUE TITLE
Pylint does not work on Windows 11

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode-pylint#18]
Testing microsoft/vscode-python#18739

I see issues in the Pylint output but this doesn't report problems for me on Windows 11.

![](https://memes.peet.io/img/22-03-85a171c4-c893-4ebc-a0ab-a8a95cc94b7b.png)

**Comment by connor4312** [comment]
I don't see any additional logs when adding that setting, but I do if I set `python.trace.server` instead. I captured those here: https://gist.github.com/connor4312/f9c0049b0c1073992fb835cf2eb2d458

Also, when looking at output channels I happened to notice this error. Unsure if related:

![image](https://user-images.githubusercontent.com/2230985/159806051-726bf953-1aca-4680-9c49-773fa6d5a490.png)

**Comment by karthiknadig** [comment]
It seems like I am not able to get the LSP to trace no matter what I set. @dbaeumer I will certainly need some input to figure out why the trace is not printing out.

**Comment by dbaeumer** [comment]
@karthiknadig how do you create the client and which settings value are you using to enable tracing?

**Comment by karthiknadig** [comment]
This is how I am creating the client:
```ts
export async function createLinterServer(
    interpreter: string,
    serverName: string,
    outputChannel: OutputChannel,
    initializationOptions: ILinterInitOptions,
): Promise<LanguageClient> {
    const serverOptions: ServerOptions = {
        command: interpreter,
        args: [LINTER_SCRIPT_PATH],
    };

    // Options to control the language client
    const clientOptions: LanguageClientOptions = {
        // Register the server for python documents
        documentSelector: [
                  { scheme: 'file', language: 'python' },
                  { scheme: 'untitled', language: 'python' },
              ],
        outputChannel: outputChannel,
        traceOutputChannel: outputChannel,
        revealOutputChannelOn: RevealOutputChannelOn.Error,
        initializationOptions,
    };

    return new LanguageClient(serverName, serverName, serverOptions, clientOptions);
}
```
`createLinterServer` is called with following params, and pretty much started like this:
```ts
const outputChannel = createOutputChannel('Pylint');
context.subscriptions.push(outputChannel);
const lsClient = createLinterServer('<path to interpreter>', 'Pylint', outputChannel, {});
context.subscriptions.push(lsClient.start());
```

This is what I have in the settings:
```
"pylint.trace.server": "verbose"
```

**Comment by dbaeumer** [comment]
That actually looks correct. And there is nothing in the output channel?

**Comment by dbaeumer** [comment]
After you created the client can you write something to the output channel using the client's `error` method.

**Comment by karthiknadig** [comment]
@dbaeumer Tried `lsClient.error('Test message');` it did not write that to the output channel.

**Comment by dbaeumer** [comment]
@karthiknadig that sounds very wired. Can you debug through it (the error call) and see what happens here: https://github.com/microsoft/vscode-languageserver-node/blob/48c9f063483c78b99b988b78f517b24106be886c/client/src/common/client.ts#L3106

Is the right channel returned?

**Comment by karthiknadig** [comment]
@dbaeumer I think I did something wrong with my previous comment. It seems to be writing to the output channel when I use `lsClient.error`.

**Comment by karthiknadig** [comment]
@dbaeumer Can you point me to where you log when `"pylint.trace.server": "verbose"` is consumed? From what I can tell, LS client seems to be only passing the console logger on connection. I may not be looking at the right thing.
![image](https://user-images.githubusercontent.com/3840081/162487412-c24055e1-50f2-4cf2-93bb-42ff71a636d8.png)

**Comment by dbaeumer** [comment]
It starts here https://insiders.vscode.dev/github/microsoft/vscode-languageserver-node/blob/48c9f063483c78b99b988b78f517b24106be886c/client/src/common/client.ts#L3726

But the actual change happens here: 

https://insiders.vscode.dev/github/microsoft/vscode-languageserver-node/blob/48c9f063483c78b99b988b78f517b24106be886c/client/src/common/client.ts#L3746

**Comment by colinfang** [comment]
Pylint doesn't work for me either. Running with Windows 10 with WSL2 (VSCode on Windows, Pylint on WSL)

Pylint output in Output channel looks correct, but somehow the messages are not captured

**Comment by colinfang** [comment]
> Pylint doesn't work for me either. Running with Windows 10 with WSL2 (VSCode on Windows, Pylint on WSL)
> 
> Pylint output in Output channel looks correct, but somehow the messages are not captured

It worked now with the latest version
